language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/common/constants/CommandConstants.java

```diff
@@ -131,4 +131,8 @@
     public static final String REPORT_USAGE_STATISTICS_COMMAND_TITLE = "Report usage statistics";
     
     public static final String CONVERT_FUNCTION_TO_PUBLIC = "Convert to public function";
+    
+    public static final String CREATE_ON_FAIL_CLAUSE = "Create on fail clause";
+
+    public static final String CREATE_ON_FAIL_WITH_DO = "Add do statement and on fail clause";
```

Comment:
```suggestion
    public static final String SURROUND_WITH_DO_ON_FAIL = "Surround with do/on-fail";
```
WDYT? @IMS94 @nadeeshaan @pcnfernando 
![Screenshot from 2021-07-13 11-12-08](https://user-images.githubusercontent.com/46120162/125397207-3f039e00-e3cb-11eb-836f-e7398290eb6c.png)

Reply:
+1. This is better